server/services/Validator.js

```diff
@@ -0,0 +1,14 @@
+const Validator = require('validatorjs');
+const { User } = require('./../models');
+
+//register custom rules 
+// Unique Email
+Validator.registerAsync('unique', async (email, attribute, req, passes) => {
+  const user = await User.where({ email }).count();
```

Comment:
We need to check email in lower case.

Reply:
yes, you are right, I used to do this before, but I changed my logic and forgot about it, thx